recipes-wolfssl/wolfssl/fips-ready/files/README.md

```diff
@@ -0,0 +1,12 @@
+# Directory for wolfSSL FIPS-Ready Bundles
+
+## Overview
+
+This directory is designated for placing the downloaded FIPS-Ready bundled from the wolfssl [download page](https://www.wolfssl.com/download/).
```

Comment:
`bundled` should be `bundle`.  Please capitalize `wolfssl` in `from the wolfSSL [download page]`.